docs/events.md

```diff
@@ -8,13 +8,14 @@ There are a Reason meetups all around the world, among others:
 - [Chicago](https://www.meetup.com/Chicago-ReasonML/)
 - [Copenhagen](https://www.meetup.com/ReasonML-CPH) ([Twitter](https://twitter.com/ReasonMLCPH))
 - [Göteborg](https://www.meetup.com/got-lambda/)
-- [Lisbon](https://www.meetup.com/ReasonML-Lisbon)
 - [Montreal](https://www.meetup.com/ReasonMTL/)
+- [New York](https://twitter.com/nycreasonml)
 - [Paris](https://www.meetup.com/ReasonML-Paris/)
-- [Phoenix](https://www.meetup.com/Phoenix-Reason)
+- [Portland](https://twitter.com/ReasonPDX)
```

Comment:
Yes, it looks sassy but didn't want to remove stuff that people put in

Reply:
https://www.meetup.com/Phoenix-Reason has been defunct for awhile now. I think it can be safely removed for the time being unless I find some more time to spin it up again.